### PR TITLE
Use new stellar notifications

### PR DESCRIPTION
**Pull request by buoyad**
Gets rid of our old handlers to use data from the notifications themselves. Everything on the account page should still update in real time.

Rough stellar RPC deltas doing a single send (counting responses)
before:
out: 28
in: 35

after:
out: 3
in: 10

r? @keybase/react-hackers cc @patrickxb

### CODE REVIEW

**Comment by buoyad** [comment]
There'll be another part to reduce `getDisplayCurrency` calls after #15056

File: shared/actions/wallets.js [hunk 1]
```diff
@@ -98,6 +98,7 @@ const openSendRequestForm = (state: TypedState, action: WalletsGen.OpenSendReque
           WalletsGen.createSetBuildingCurrency({
             currency:
               action.payload.currency ||
+              (state.wallets.lastSentXLM && 'XLM') ||
```
Comment (chrisnojima) [inline shared/actions/wallets.js]: === ?

File: shared/reducers/wallets.js [hunk 2]
```diff
@@ -53,6 +53,22 @@ export default function(state: Types.State = initialState, action: WalletsGen.Ac
...
+      )
+    case WalletsGen.recentPaymentsReceived:
+      const newPayments = I.Map(action.payload.payments.map(p => [p.id, Constants.makePayment().merge(p)]))
+      return state
+        .updateIn(['paymentsMap', action.payload.accountID], (paymentsMap = I.Map()) =>
+          paymentsMap.merge(newPayments)
+        )
+        .updateIn(
+          ['paymentCursorMap', action.payload.accountID],
+          cursor => cursor || action.payload.paymentCursor
```
Comment (chrisnojima) [inline shared/reducers/wallets.js]: why keep an existing cursor?

File: shared/actions/wallets.js (hunk 1 above)
Comment (buoyad) [reply shared/actions/wallets.js]: thanks!

Comment (buoyad) [reply shared/actions/wallets.js]: whoops actually I thought that this was a different file. not a typo! if there isn't an `action.payload.currency` and we know `lastSentXLM`, this sets the currency to `XLM`.

File: shared/reducers/wallets.js (hunk 2 above)
Comment (buoyad) [reply shared/reducers/wallets.js]: in case we've already loaded payments

File: shared/actions/wallets.js (hunk 1 above)
Comment (chrisnojima) [reply shared/actions/wallets.js]: k

**Pull request merged**